CONTRIBUTING.md

```diff
@@ -59,6 +59,19 @@ run local without the need of CI. Have a look at the
 [Testing](docs/getting-started.md#testing)
 section in the [Developer Guide](docs/getting-started.md).
 
+### Signing your commits
+
+All your commits need to be signed in order to get merged. In short the commit message
+needs to have a line looking like this:
+
+```
+Signed-off-by: John Doe <jdoe@example.org>
+```
+
+This can be done by adding [`--signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to your git command line.
+
+Please see [here](https://groups.google.com/forum/#!msg/kubevirt-dev/VdFxY6mXAH4/ftw8yMl6AAAJ) for further details.
```

Comment:
Instead of explaining what to do technically, I guess we should say what people are actually signing. I think that this link explains that we need to enable it for CNCF, but not what people are signing.

Reply:
Good point. I'll try to explain in short what signing means and give context on that.

Reply:
Done.